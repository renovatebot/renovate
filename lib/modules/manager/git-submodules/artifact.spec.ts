import { updateArtifacts } from '.';

describe('modules/manager/git-submodules/artifact', () => {
  describe('updateArtifacts()', () => {
    it('returns empty content', () => {
      expect(
        updateArtifacts({
          packageFileName: '',
          updatedDeps: [{ depName: '' }],
          newPackageFileContent: '',
          config: {},
        })
      ).toMatchObject([{ file: { type: 'addition', path: '', contents: '' } }]);
    });

    it('returns two modules', () => {
      expect(
        updateArtifacts({
          packageFileName: '',
          updatedDeps: [{ depName: 'renovate' }, { depName: 'renovate-pro' }],
          newPackageFileContent: '',
          config: {},
        })
      ).toMatchObject([
        { file: { type: 'addition', path: 'renovate', contents: '' } },
        { file: { type: 'addition', path: 'renovate-pro', contents: '' } },
      ]);
    });
  });
});
