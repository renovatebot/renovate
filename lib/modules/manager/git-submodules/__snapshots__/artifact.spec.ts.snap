// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`modules/manager/git-submodules/artifact updateArtifacts() returns empty content 1`] = `
[
  {
    "file": {
      "contents": "",
      "path": "",
      "type": "addition",
    },
  },
]
`;

exports[`modules/manager/git-submodules/artifact updateArtifacts() returns two modules 1`] = `
[
  {
    "file": {
      "contents": "",
      "path": "renovate",
      "type": "addition",
    },
  },
  {
    "file": {
      "contents": "",
      "path": "renovate-pro",
      "type": "addition",
    },
  },
]
`;
